ee/tabby-ui/app/chat/page.tsx

```diff
@@ -152,14 +166,7 @@ export default function ChatPage() {
         event.altKey == focusKeyInfo.altKey &&
         event.shiftKey == focusKeyInfo.shiftKey
       ) {
-        if (
-          chatInputRef.current &&
-          document.activeElement !== chatInputRef.current
-        ) {
-          chatInputRef.current.focus()
-        } else {
-          server?.focusOnEditor()
-        }
```

Comment:
given command-l focus works as expected, I feel there's no way to have additional keyboard support for switching to inputRef